test/support/integration/enterprise_server.ex

```diff
@@ -0,0 +1,186 @@
+defmodule LivebookTest.Integration.EnterpriseServer do
+  @moduledoc false
+  use GenServer
+
+  defmodule WrappedCollectable do
```

Comment:
```suggestion
  defmodule CollectableMapper do
```

Reply:
Alternatively, since we we have a specific use case, we can define `Callback.new(fun)` that is a collectable wrapping the given function.

Reply:
How would it work?